spec/services/saml_request_validator_spec.rb

```diff
@@ -176,7 +192,33 @@
         SamlRequestValidator.new.call(
           service_provider: sp,
           authn_context: authn_context,
-          nameid_format: 'urn:oasis:names:tc:SAML:2.0:nameid-format:persistent',
+          nameid_format: name_id_format,
+        )
+
+        expect(FormResponse).to have_received(:new).
+          with(success: false, errors: errors, extra: extra)
+      end
+    end
+
+    context 'valid authn context and sp and unauthorized nameID format' do
+      it 'returns FormResponse with success: true' do
```

Comment:
Looks like a copy paste error here. I think we mean `success: false`.

Reply:
Yes, already fixed